test/Atmos/EDMF/stable_bl_edmf_incompressible.jl

```diff
@@ -0,0 +1,260 @@
+using JLD2, FileIO
```

Comment:
We need to add this to buildkite

Reply:
Do we need it now that we are using the netcdf diagnostics?

Reply:
I just mean the driver file (`test/Atmos/EDMF/stable_bl_edmf_incompressible.jl`)